src/store/index.js

```diff
@@ -0,0 +1,19 @@
+import { createStore, applyMiddleware } from "redux";
+import createSagaMiddleware from "redux-saga";
+import rootreducer from "./rootReducer";
+import { composeWithDevTools } from "redux-devtools-extension";
+
+import rootSaga from "../sagas";
+
+const configureStore = () => {
```

Comment:
Add descriptive comments to this function.

Reply:
Okay, let me look into this.